### ISSUE TITLE
interacting with components should be consistent

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#180725]
It would be best to have a consistent experience across components so that less learning and trial/error are needed - in particular for screen reader users. @jooyoungseo found the following discrepancies: 

Aligning the keybinding used to focus an input box:
- In the search editor, navigate in the results.  `ctrl + down/up arrow` to refocus the input
- In the keybindings editor, navigate in the results. `ctrl+f` refocuses the input is used. 
- Should we support both keybindings?

`View: Focus into Primary Side Bar` 
- Explorer view:
   - `ctrl+shift+e` focuses the explorer view
   - navigate the results
   - `ctrl+shift+e` is applied again (focusing the editor)
   - `ctrl+shift+e` focuses the correct item in the list
- Extensions view:
   - `ctrl+shift+x` focuses the extension view
   - navigate the results
   - `ctrl+shift+x` is applied again (focusing the editor)
   - `View: Focus into Primary Side Bar` does not refocus where I was in the extensions view. 
- Search view:
   - `ctrl+shift+f` focuses the search view
   - navigate the results and hit `enter` on one of them
   - `View: Focus into Primary Side Bar` refocuses the correct search result
   - `ctrl+shift+f` does not toggle back and forth between the editor like it does for the explorer/extensions view

**Comment by meganrogge** [comment]
So it seems the requests would be to:

- have `ctrl+shift+f` toggle back and forth between the editor and the search view - I'm not sure that would be expected
- remember where in the extensions view we were before leaving that so it can be refocused upon returning 

@isidorn do you have thoughts or should i loop in the area owners?

**Comment by isidorn** [comment]
For `ctrl+shift+f` - I am not convinced. None of the other focus view commands toggle. Only Ctrl+B toggles. So we should remain consistent here, and if we change `ctrl+shift+f` - we should change for all views. But I am not sure it is a big gain.

For extensions view - makes perfect sense.

**Comment by sandy081** [comment]
Is that we are just scoping this issue down to extensions only?

**Comment by isidorn** [comment]
That makes sense to me.

**Comment by VSCodeTriageBot** [comment]
We closed this issue because we [don't plan to address it](https://aka.ms/vscode-out-of-scope) in the foreseeable future. If you disagree and feel that this issue is crucial: we are happy to listen and to reconsider.

If you wonder what we are up to, please see our [roadmap](https://aka.ms/vscoderoadmap) and [issue reporting guidelines](https://aka.ms/vscodeissuereporting).

Thanks for your understanding, and happy coding!
